### ISSUE TITLE
Flag parameter for enabling/disabling AuditController

### ISSUE THREAD

**Issue by p365labs** [DamienHarper/DoctrineAuditBundle#153]
Hi all, I'm wondering how I can disable the actions, and so the routes defined inside the
AuditController. 
The FrontEnd is really useful, but at a certain point, I don't want to have that kind of route, in production, exposed. I've tried to play a bit with expression language with "condition" key for routing. something like:

```condition: '%enable_audit_controller% === true')```

where **enable_audit_controller** is a container parameter. But it's not working. This means if I don't find a method to disable those routes, they will be exposed in production.

@DamienHarper & @maxhelias any ideas?

**Comment by DamienHarper** [comment]
@p365labs If your application supports authentication, you can add a role requirement on audit routes in your `security.yaml`

```yaml
security:
    access_control:
        - { path: ^/audit, role: ROLE_ADMIN }
```

**Comment by maxhelias** [comment]
Hi!
Or you can also remove the `DH\DoctrineAuditBundle\Controller\AuditController` definition with a CompilerPass.

BTW, @DamienHarper, maybe we could add a security configuration for the `AuditController` ?

**Comment by DamienHarper** [comment]
@maxhelias yes, I was thinking about this.

**Comment by p365labs** [comment]
@DamienHarper & @maxhelias Yes got u what u mean, but I was thinking more to a sort of **feature flag**.
I'm going to describe the use case as it will be more clear: 

I'm not going to use twig as our application exposes *APIs* which are consumed by a Vue.JS SPA client. In this particular situation, it's useless for me exposing routes to the word that I have to disable (compiler pass) or with specific access (ROLE management). I just don't want to expose them.

**Comment by maxhelias** [comment]
@p365labs I think you can keep open the issue. It would be interesting from my point of view to have this feature flag

**Comment by p365labs** [comment]
@maxhelias ok I will investigate a good way to do it, during these holidays. Tnx!

**Comment by DamienHarper** [comment]
I close this issue, fixed by PR #161

**Comment by DamienHarper** [comment]
@p365labs 3.3.0 is released
